docs/connector/08-01-establish-secure-connection-with-compass.md

```diff
@@ -0,0 +1,85 @@
+# Establish a secure connection with Compass
+
+To establish a secure connection with Compass and generate the client certificate, follow this tutorial. 
+
+## Prerequisites
+
+- [OpenSSL toolkit](https://www.openssl.org/docs/man1.0.2/apps/openssl.html) to create a Certificate Signing Request (CSR), keys, and certificates which meet high security standards
+- Compass (version 1.8 or higher)
+- Registered Application
+- Runtime connected to Compass
+
+## Steps
+
+1. Get the Connector URL and the one-time token.
+
+    To get the Connector URL and the one-time token which allow you to fetch the required configuration details, use the Compass Console.
+    
+    Alternatively, make a call to the Director including the `Tenant` header with Tenant ID and `authorization` header with the Dex Bearer token. Use the following mutation: 
+    
+    ```graphql
+    mutation { 
+        result: generateOneTimeTokenForApplication(id: "{APPLICATION_ID}") { 
+            token 
+            connectorURL 
+        }
+    }
+    ```
+   
+   > **NOTE:** The one-time token expires after 5 minutes.
+
+2. Get the CSR information and configuration details from Kyma using the one-time token.
+
+    To get the CSR information and configuration details, send this GraphQL query with the one-time token included in the `connector-token` header to the Connector URL:
```

Comment:
We should emphasize the need of having a `connector-token` header in the call, it is easy to overlook that in the current form.